### PR TITLE
Handles quizzes eval for multi-value fields and other fixes

### PR DESCRIPTION
**Pull request by sheralim012**
- Handled multi-value fields result evaluation
- Fixed rendering of multiselect
- Fixed highlighting of right and wrong answers for multi-value fields
- Handled multi-value fields description
- Updated help text for correct answer field 
- Updated back button in quiz result page

### CODE REVIEW

File: questionnaires/models.py [hunk 1]
```diff
@@ -663,16 +663,26 @@ def get_context(self, request, *args, **kwargs):
             form_class = self.get_form_class()
             form = form_class(data=request.POST, page=self, user=request.user)
 
-            for k, v in form.fields.items():
-                form.fields[k].widget.attrs['readonly'] = True
+            # for k, v in form.fields.items():
```
Comment (ehmadzubair) [inline questionnaires/models.py:666]: remove this if not needed

File: questionnaires/models.py [hunk 2]
```diff
@@ -663,16 +663,26 @@ def get_context(self, request, *args, **kwargs):
...
+            #     form.fields[k].widget.attrs['readonly'] = True
```
Comment (ehmadzubair) [inline questionnaires/models.py:668]: Remove this if not needed

File: questionnaires/models.py [hunk 3]
```diff
@@ -663,16 +663,26 @@ def get_context(self, request, *args, **kwargs):
...
+                    correct_answer = field.correct_answer.split(',')
+                else:
+                    correct_answer = field.correct_answer
+
+                if field.field_type == 'checkbox':
+                    answer = form_data.get(field.clean_name, 'off')
+                else:
+                    answer = form_data.get(field.clean_name)
+
+                is_correct = answer == correct_answer
```
Comment (ehmadzubair) [inline questionnaires/models.py:685]: create a set() and then compare the answers

File: questionnaires/models.py (hunk 2 above)
Comment (sheralim012) [reply questionnaires/models.py]: done

File: questionnaires/models.py (hunk 3 above)
Comment (sheralim012) [reply questionnaires/models.py]: done

**Review by ehmadzubair — APPROVED** [review]

**Pull request merged**